README.md

```diff
@@ -1,123 +1,96 @@
+Juju Box is a docker container project that provides a configured Juju
+environment with all the tools you need to work with 
+[Juju](https://jujucharms.com).
 
-Juju provisions machines, deploy services and orchestrates them into
-systems.
+Juju is an open source, universal model for applications. Juju allows you to 
+deploy, configure, manage, maintain, and scale cloud services quickly and 
+efficiently on public clouds, as well as on physical servers.
 
-This container comes with the basic tools you need to get started w/
-juju.
+This container comes with the basic tools you need to get started with Juju.
 
  - juju
- - juju quickstart
- - [juju plugins](https://github.com/juju/plugins)
+ - juju-deployer
 
-Note: jujubox does not currently install juju local provider.  We hope
-to roll something that works like local provider in a container
-solution soon.
+The [charmbox](https://github.com/juju-solutions/charmbox) container starts 
+jujubox and adds the functionality of 
+[charm-tools](https://github.com/juju/charm-tools) and 
+[bundletester](https://github.com/juju-solutions/bundletester).
 
-# Install
+# Install Docker
 
-### Linux
-Install Docker using the package manager for your distro, or
-[get.docker.com](https://get.docker.com/).
+## Linux
+Install Docker using the package manager for your Linux distribution, or
+[get.docker.com](https://get.docker.com/). We recommend Docker version 1.4.1 or
+greater.
 
-Docker >= 1.4.1 recommended.
+## Mac OS X
 
-### OS X
+There are several ways to 
+[install Docker on Mac OS X](https://docs.docker.com/engine/installation/mac/). 
+Consult the latest documentation on
+[Getting Started with Docker for Mac](https://docs.docker.com/docker-for-mac/).
 
-Docker does not run natively in OS X, so we'll need to use VirtualBox
-and boot2docker. This is relatively easy for users of brew:
+# Download jujubox
 
-```
-brew install cask
-brew cask install virtualbox
-brew install docker
-brew install boot2docker
-
-boot2docker init
-boot2docker up
-```
-
-When boot2docker finishes, it will prompt you to export Docker
-environment variables. With that done, you're ready to use docker.
-You do not need to run the docker commands using sudo because
-boot2docker runs as root.
-
-    docker run  -ti jujusolutions/jujubox
-
-# Download
-
-Download the Juju Box source from github or the container from
-[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/):
-
-### Linux
-You can clone the repository from github.
+If you want to see the code you can download the jujubox project from 
+[github](https://github.com/juju-solutions/jujubox) and build it from source.
 
 ```
 git clone https://github.com/jujusolutions/jujubox.git && cd jujubox
-
+docker build -t jujusolutions/jujubox ./
 ```
 
-Or download and install the container directly from hub.docker.io:
+Or simply pull the jujubox container from
+[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/).
```

Comment:
Maybe instead of keeping the hub.docker.io domain which goes nowhere, you should change this to read  "the docker hub" and keep that link.